web/lib/supabase/reactions.ts

```diff
@@ -24,13 +28,13 @@ export async function getLikedContracts(userId: string) {
 }
 
 export async function getLikedContractsCount(userId: string) {
-  const { data } = await run(
+  const { count } = await run(
     db
       .from('user_reactions')
-      .select('count')
+      .select('*', { head: true, count: 'exact' })
```

Comment:
What's the reasoning behind the change here? I thought selecting 'count' worked fine.

Reply:
The way this Javascript library works, is it looks at the string in your `select` to figure out what the return type of your query is, by comparing it against the database schema (i.e. what rows are in the table and what the types of the rows are.) Since `count` isn't a row, it gets confused and spits out a type error.

(Selecting `count` is also very quirky -- I had no idea that was even legal PGSQL until I saw you do it. The normal way would be to select `count(*)`, which is what this Javascript will end up producing.)